### PR TITLE
Debug window with FPS

### PR DESCRIPTION
**Pull request by blexaer**
## Motivation and Context
My fan spins like a turbine after a few minutes, which prompted me to check what the cause was.

## How Has This Been Tested?
Window displays frames per second.

## Screenshots (if appropriate):
[Screenshot](https://i.imgur.com/qvfpK0S.png)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: client/game.cpp [hunk 1]
```diff
@@ -129,6 +129,18 @@ void Game::Update(double delta, double mouse_x, double mouse_y, int window_width
...
+	// Franes per second
+	//
+	// If one second has passed, we have the number of frames in that second, giving us frames per second
+	frame_deltas += delta;
+	frames++;
+	if (frame_deltas >= 1) {
+		fps = frames;
+		frame_deltas = 0;
+		frames = 0;
+		//std::cout << std::to_string(fps) << std::endl;
```
Comment (adam4813) [inline client/game.cpp:141]: Debug line?

File: client/game.hpp [hunk 2]
```diff
@@ -42,11 +42,18 @@ class Game {
 
 	std::shared_ptr<LuaScript> config_script;
 
+	// Frames per second
+	int fps;
```
Comment (adam4813) [inline client/game.hpp:46]: Should this be unsigned?

**Review by adam4813 — APPROVED** [review]
A few notes, but seems alright to me.

File: client/game.cpp (hunk 1 above)
Comment (blexaer) [reply client/game.cpp:141]: initially just went to console, removing

File: client/game.hpp (hunk 2 above)
Comment (blexaer) [reply client/game.hpp:46]: Sure

**Pull request merged**